pyproject.toml

```diff
@@ -228,3 +228,9 @@ default-groups = [
 [[tool.uv.index]]
 name = "nvidia-pypi"
 url = "https://pypi.nvidia.com"
+
+[tool.uv.sources]
+llama-index-core = {workspace = true}
+
+[tool.uv.workspace]
+members = ["llama-index-core", "llama-index-integrations/workflows/*"]
```

Comment:
temporary change to use workspaces (to just link the core source to the package), to be reverted (but maybe eventually want to do this?). The shared dependency versions would need to be resolved to the same/compatible versions